recipes-collection/components/RecipeImage/Display.tsx

```diff
@@ -5,5 +5,6 @@ export function RecipeImageDisplay({
 }: {
   image: StaticImageProps;
 }) {
+  // eslint-disable-next-line @next/next/no-img-element, jsx-a11y/alt-text
   return <img {...props} />;
 }
```

Comment:
We intentionally sidestep `Image` here, and `alt` should come from the wrapping call.